src/components/Tab/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+import Button, { ButtonProps } from '../Button';
+import List from '../List';
+
+import styles from './styles.css';
+
+export interface TabProps {
+    children: React.ReactNode;
+}
+
+const tabContext = React.createContext({
```

Comment:
```suggestion
const TabContext = React.createContext({
```

Reply:
or we should destructure tabContext before using it in jsx

Bad:
         <tabContext.Provider value={contextValue}> 
Good:
        <Provider value={contexValue}>